src/lib/firestoreController.ts

```diff
@@ -42,6 +43,7 @@ class FirestoreController {
       .then((res) => res.json())
```

Comment:
I think it's a good idea to capture an exception with the `publicKey` and email / firebase user id when `{indexer}/accounts` returns an empty array here and other relevant places we call it wdyt?